test/libsolidity/syntaxTests/controlFlow/storageReturn/emptyReturn_fine.sol

```diff
@@ -1,7 +0,0 @@
-contract C {
-    struct S { bool f; }
-    S s;
-    function f() internal view returns (S storage c, S storage d) { c = s; d = s; return; }
-    function g() internal view returns (S storage, S storage) { return (s,s); }
```

Comment:
This one could stay

Reply:
Yes, but it shouldn't be in a file named ``emptyReturn_file.sol``, though... I'll add it back with a different name.